index.html

```diff
@@ -192,7 +194,7 @@ <h3 id="selectHardwareAcceleration">Hardware Acceleration</h3>
 	                                    <div class="col-md-9 r-content pr-0 pl-md-4" role="listbox" id="listbox-4" aria-labelledby="selectHardwareAcceleration" aria-describedby="decriptionHardwareAcceleration">
 	                                        <div class="row hardwareAcceleration">
 	                                            <div class="col-lg-2dot5 col r-option version selected" role="option" tabindex="0" aria-selected="true" id="DefaultCPU">
-	                                                <span>Default <abbr>CPU</abbr></span></div>
+	                                                <span>Default&nbsp; <abbr>CPU</abbr></span></div>
```

Comment:
is this change intentional?

Reply:
> is this change intentional?
yes, looks a bit cleaner to separate the words "Default CPU" vs DefaultCPU since it's a conceptual term